rdr_service/model/__init__.py

```diff
@@ -87,5 +88,6 @@
     ('rdr_service.model.bq_genomics', 'BQGenomicSetMemberView'),
     ('rdr_service.model.bq_genomics', 'BQGenomicJobRunView'),
     ('rdr_service.model.bq_genomics', 'BQGenomicGCValidationMetricsView'),
-    ('rdr_service.model.bq_genomics', 'BQGenomicFileProcessedView')
+    ('rdr_service.model.bq_genomics', 'BQGenomicFileProcessedView'),
+    ('rdr_service.model.bq_genomics', 'BQGenomicManifestFileView')
```

Comment:
Does `BQGenomicManifestFeedback` need to be added here too?

Reply:
Thanks for catching that. Adding them now.